uhi/typing/plottable.py

```diff
@@ -97,39 +97,50 @@ class PlottableHistogram(Protocol):
 
     def values(self) -> ArrayLike:
         """
-        Values returns the counts for simple histograms or the mean for mean
-        histograms.  If this has a "mean" interpretation, then values is
-        undefined and left up to the Producer implementation if counts is less
-        than 1. A Consumer should check counts before using this.
-
+        Returns the accumulated values. The counts for simple histograms, the
+        sum of weights for weighted histograms, the mean for profiles, etc.
+ 
+        If counts is equal to 0, the value in that cell is undefined if
+        kind == "MEAN".
+        
         All methods can have a flow=False argument - not part of this Protocol.
         If this is included, it should return an array with flow bins added,
         normal ordering.
         """
 
     def variances(self) -> Optional[ArrayLike]:
         """
-        Variance returns the variance of the values (for "MEAN" histograms,
-        this is not the variance of the counts, but the variance of the mean).
-        If unweighed, this returns None.
+        Returns the estimated variance of the accumulated values. The sum of squared 
+        weights for weighted histograms, the variance of samples for profiles, etc.
+        For an unweighed histogram, this returns the same as values.
```

Comment:
```suggestion
        For an unweighed histogram where kind == "COUNT", this returns the same
        as values if the histogram was not filled with weights, and None otherwise.
```

Reply:
@HDembinski I believe this is the change I implemented. Maybe the wording should be slightly more permissive? I think it's pretty close, though.